src/frontend/routes/index.js

```diff
@@ -46,17 +49,20 @@ export const NOTFOUND = '*';
 export const availableRoutes = (
   <Routes>
     <Route exact path={LANDING} element={<Landing />} />
-    <Route exact path={HOMEPAGE} element={<Homepage />} />
     <Route exact path={MOCKMAN} element={<MockAPI />} />
-    <Route exct path={POST} element={<IndividualPost />} />
-    <Route exct path={EXPLORE} element={<ExploreFeed />} />
-    <Route exct path={BOOKMARK} element={<Bookmark />} />
-    <Route exct path={NOTIFICATION} element={<Notification />} />
-    <Route exct path={PROFILE} element={<Profile />} />
-    <Route exct path={USERPROFILE} element={<AnyUserProfile />} />
-    <Route exct path={SETTINGS} element={<Settings />} />
     <Route exct path={SIGNIN} element={<Signin />} />
     <Route exct path={SIGNUP} element={<Signup />} />
     <Route path={NOTFOUND} element={<NotFound />} />
+
+    <Route exact path={LANDING} element={<PrivateRoute />}>
+      <Route exact path={HOMEPAGE} element={<Homepage />} />
+      <Route exct path={POST} element={<IndividualPost />} />
+      <Route exct path={EXPLORE} element={<ExploreFeed />} />
+      <Route exct path={BOOKMARK} element={<Bookmark />} />
+      <Route exct path={NOTIFICATION} element={<Notification />} />
+      <Route exct path={PROFILE} element={<Profile />} />
+      <Route exct path={USERPROFILE} element={<AnyUserProfile />} />
+      <Route exct path={SETTINGS} element={<Settings />} />
```

Comment:
I think there is typo in `exact`. Check this once

Reply:
thanks for pointing that out